.github/workflows/ci.yml

```diff
@@ -61,8 +52,20 @@ jobs:
           macos: py312-test
           posargs: -v
 
-        - name: Python 3.12 with coverage
+        - name: Python 3.12 (Linux) with coverage
           linux: py312-test-cov
           posargs: -v
           coverage: codecov
 
+        - name: Python 3.12 (Windows with dev dependencies)
+          windows: py312-test-devdeps
+          posargs: -v
+
+        - name: Python 3.12 (OSX with dev dependencies)
+          macos: py312-test-devdeps
+          posargs: -v
+
```

Comment:
This is overkill. Linux is enough for devdeps.

```suggestion
```

Reply:
It's OK. It's a small package